dds/DCPS/Serializer.cpp

```diff
@@ -196,6 +196,13 @@ bool EncapsulationHeader::to_encoding(
   return true;
 }
 
+static const int FOUR_BYTE_ALIGNMENT = 4;
```

Comment:
Could this be one of the existing constants in Serializer.h?  Maybe `ALIGN_XCDR2`?

Reply:
The padding marker is not specific to any encoding, so I don't think we should use `ALIGN_XCDR2`.

Reply:
If it's going to be a constant outside a function, it should probably go in `EcapsulationHeader`.